src/ckeditor.js

```diff
@@ -12,9 +14,56 @@
  * @singleton
  */
 
-CKEDITOR.define( function() {
+CKEDITOR.define( [ 'editor', 'mvc/collection', 'promise' ], function( Editor, Collection, Promise ) {
 	var CKEDITOR = {
 		/**
+		 * A collection containing all editor instances created.
+		 */
+		instances: new Collection(),
+
+		/**
+		 * Creates an editor instance for the provided DOM element.
+		 *
+		 * The creation of editor instances is an asynchronous operation, therefore a promise is returned by this
+		 * method.
+		 *
+		 *     CKEDITOR.create( '#content' );
+		 *
+		 *     CKEDITOR.create( '#content' ).then( function( editor ) {
+		 *         // Manipulate "editor" here.
+		 *     } );
+		 *
+		 * @param {String|Object} element An element selector or a DOM element, which will be the source for the
+		 * created instance.
+		 * @returns {Promise} A promise, which will be fulfilled with the created editor.
+		 */
+		create: function( element ) {
+			var that = this;
+
+			return new Promise( function( resolve, reject ) {
+				// If a query selector has been passed, transform it into a real element.
+				if ( typeof element == 'string' ) {
+					element = document.querySelector( element );
```

Comment:
I agree with @Reinmar... still create() should accept a selector, because this is one of the most basic features. It should create one and only element, just like implemented right now.

Then, one can still interact through elements like you exemplified and have the problem solved. And in the future, if really needed, something like @adelura proposed is interesting... but in the opposite way, by introducing a `createAll()`.

Anyway, for now, as I commented already, let's leave this discussion as a followup ticket for the PR.